test/parallel/test-tls-keylog-tlsv13.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+
+const common = require('../common');
+if (!common.hasCrypto)
+  common.skip('missing crypto');
+
+const assert = require('assert');
+const events = require('events');
+const tls = require('tls');
+const fixtures = require('../common/fixtures');
+
+async function main() {
+  // Create server / client pair
+  const server = tls.createServer({
+    key: fixtures.readSync('/keys/agent2-key.pem'),
+    cert: fixtures.readSync('/keys/agent2-cert.pem'),
+    minVersion: 'TLSv1.3',
+    maxVersion: 'TLSv1.3',
+  });
+  server.listen();
+  await events.once(server, 'listening');
+  server.once('connection', () => server.close());
+
+  const client = tls.connect({
+    port: server.address().port,
+    rejectUnauthorized: false,
+  });
+
+  // Record keylog lines
+  const lines = { client: new Set(), server: new Set() };
+  client.on('keylog', (line) => lines.client.add(line));
```

Comment:
I think this test could be simpler, and as useful, if it used common.mustCall() here, rather than collecting sets and comparing them later.

Reply:
I'm not sure I understand... keylog must be emitted exactly 5 times (5 secrets) with the same output on both sides, but the order may differ. That's why I collect them in sets and later do an unordered comparison. How would you implement it?

Reply:
I've found code that collects things in events and then does comparisons later to be harder to maintain, but this code as-is is pretty clear, so I'm OK with it as is. That said, I would be OK with a `common.mustCall(.., 5)` and no comparison, the identicallity seems like a test of the OpenSSL API, as opposed to node's. If the session secrets are generated in the same order on both sides, the cb on both sides could be `(line) => { if(!saved)  return saved = line; assert(saved === line); saved = null}`.

Reply:
Good point, I can't really think of a situation where the identicallity test would reveal a bug in Node's side... I'm only testing for 5 calls now, marking this as resolved.